Makefile

```diff
@@ -135,6 +135,9 @@ test-e2e: build ## Run end-to-end tests using kuttl (https://kuttl.dev/)
 	@[ "${AIVEN_PROJECT_NAME}" ] || ( echo ">> variable AIVEN_PROJECT_NAME is not set"; exit 1 )
 	kubectl kuttl test --config test/e2e/kuttl-test.yaml
 
+test-e2e-preinstalled:
```

Comment:
Ok, now I have something wrong with the web hooks:
```
  logger.go:42: 14:35:38 | projectvpc-ref/1-resources | ProjectVPC:kuttl-test-talented-antelope/k8s-e2e-projectvpc-ref created
=== CONT  kuttl/harness/mysql-simple-cluster
    case.go:361: failed in step 1-mysql
    case.go:363: Internal error occurred: failed calling webhook "mmysql.kb.io": failed to call webhook: Post "https://aiven-operator-webhook-service.default.svc:443/mutate-aiven-io-v1alpha1-mysql?timeout=10s": dial tcp 10.96.173.91:443: connect: connection refused
=== CONT  kuttl/harness/projectvpc-ref
    case.go:361: failed in step 1-resources
    case.go:363: Internal error occurred: failed calling webhook "mkafka.kb.io": failed to call webhook: Post "https://aiven-operator-webhook-service.default.svc:443/mutate-aiven-io-v1alpha1-kafka?timeout=10s": dial tcp 10.96.173.91:443: connect: connection refused
    case.go:363: Internal error occurred: failed calling webhook "mopensearch.kb.io": failed to call webhook: Post "https://aiven-operator-webhook-service.default.svc:443/mutate-aiven-io-v1alpha1-opensearch?timeout=10s": dial tcp 10.96.173.91:443: connect: connection refused
    case.go:363: Internal error occurred: failed calling webhook "mredis.kb.io": failed to call webhook: Post "https://aiven-operator-webhook-service.default.svc:443/mutate-aiven-io-v1alpha1-redis?timeout=10s": dial tcp 10.96.173.91:443: connect: connection refused
=== CONT  kuttl/harness/cassandra-simple-cluster
    logger.go:42: 14:35:40 | cassandra-simple-cluster | Failed to collect events for cassandra-simple-cluster in ns kuttl-test-literate-gelding: no matches for kind "Event" in version "events.k8s.io/v1beta1"
    logger.go:42: 14:35:40 | cassandra-simple-cluster | Trying with events eventsv1 API...
    logger.go:42: 14:35:40 | cassandra-simple-cluster | cassandra-simple-cluster events from ns kuttl-test-literate-gelding:
=== CONT  kuttl/harness/clickhouse-postgresql
    case.go:361: failed in step 1-clickhouse
    case.go:363: Internal error occurred: failed calling webhook "mclickhouse.kb.io": failed to call webhook: Post "https://aiven-operator-webhook-service.default.svc:443/mutate-aiven-io-v1alpha1-clickhouse?timeout=10s": dial tcp 10.96.173.91:443: connect: connection refused
    case.go:363: Internal error occurred: failed calling webhook "mpg.kb.io": failed to call webhook: Post "https://aiven-operator-webhook-service.default.svc:443/mutate-aiven-io-v1alpha1-postgresql?timeout=10s": dial tcp 10.96.173.91:443: connect: connection refused
=== CONT  kuttl/harness/mysql-simple-cluster
    logger.go:42: 14:35:40 | mysql-simple-cluster | Failed to collect events for mysql-simple-cluster in ns kuttl-test-whole-racer: no matches for kind "Event" in version "events.k8s.io/v1beta1"
    logger.go:42: 14:35:40 | mysql-simple-cluster | Trying with events eventsv1 API...
    logger.go:42: 14:35:40 | mysql-simple-cluster | mysql-simple-cluster events from ns kuttl-test-whole-racer:
=== CONT  kuttl/harness/projectvpc-ref
    logger.go:42: 14:35:41 | projectvpc-ref | Failed to collect events for projectvpc-ref in ns kuttl-test-talented-antelope: no matches for kind "Event" in version "events.k8s.io/v1beta1"
    logger.go:42: 14:35:41 | projectvpc-ref | Trying with events eventsv1 API...
=== CONT  kuttl/harness/cassandra-simple-cluster
    logger.go:42: 14:35:41 | cassandra-simple-cluster | Deleting namespace: kuttl-test-literate-gelding
=== CONT  kuttl/harness/clickhouse-postgresql
    logger.go:42: 14:35:43 | clickhouse-postgresql | Failed to collect events for clickhouse-postgresql in ns kuttl-test-positive-camel: no matches for kind "Event" in version "events.k8s.io/v1beta1"
    logger.go:42: 14:35:43 | clickhouse-postgresql | Trying with events eventsv1 API...
    logger.go:42: 14:35:43 | clickhouse-postgresql | clickhouse-postgresql events from ns kuttl-test-positive-camel:
=== CONT  kuttl/harness/projectvpc-ref
    logger.go:42: 14:35:43 | projectvpc-ref | projectvpc-ref events from ns kuttl-test-talented-antelope:
=== CONT  kuttl/harness/mysql-simple-cluster
    logger.go:42: 14:35:43 | mysql-simple-cluster | Deleting namespace: kuttl-test-whole-racer
=== CONT  kuttl/harness/clickhouse-postgresql
    logger.go:42: 14:35:43 | clickhouse-postgresql | Deleting namespace: kuttl-test-positive-camel
=== CONT  kuttl/harness/projectvpc-ref
    logger.go:42: 14:35:43 | projectvpc-ref | Deleting namespace: kuttl-test-talented-antelope
=== CONT  kuttl
    harness.go:399: run tests finished
    harness.go:508: cleaning up
    harness.go:563: removing temp folder: ""
--- FAIL: kuttl (16.33s)
    --- FAIL: kuttl/harness (0.00s)
        --- FAIL: kuttl/harness/clickhouse-simple-cluster (11.79s)
        --- FAIL: kuttl/harness/redis-simple-cluster (11.81s)
        --- FAIL: kuttl/harness/opensearch-simple-cluster (11.82s)
        --- FAIL: kuttl/harness/grafana-simple-cluster (11.82s)
        --- FAIL: kuttl/harness/kafka-simple-cluster (11.83s)
        --- FAIL: kuttl/harness/kafka-logs (11.84s)
        --- FAIL: kuttl/harness/connect-kafka-to-postgres (11.88s)
        --- FAIL: kuttl/harness/simple-kafka-topic (11.88s)
        --- FAIL: kuttl/harness/cassandra-simple-cluster (3.03s)
        --- FAIL: kuttl/harness/mysql-simple-cluster (4.45s)
        --- FAIL: kuttl/harness/clickhouse-postgresql (4.47s)
        --- FAIL: kuttl/harness/projectvpc-ref (4.48s)
FAIL
make: *** [Makefile:139: test-e2e-preinstalled] Error 1
```

Reply:
Well, I can't say why it doesn't work without linux machine. Thanks for testing.